code-challenges.test.js

```diff
@@ -134,20 +138,20 @@ const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
-// describe("allCaps",() => {
-//     it("takes in an array of words and returns an array with all the words capitalized.", () => {
-//         expect (allCaps(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
-//         expect (allCaps(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
-//          })
-//         })
+describe("allCaps",() => {
+    it("takes in an array of words and returns an array with all the words capitalized.", () => {
+        expect (allCaps(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+        expect (allCaps(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+         })
+        })
        
 // // b) Create the function that makes the test pass.
 
-// const allCaps = (array) => {
-//    return array.map((string) => { 
-//    return string.charAt(0).toUpperCase() + string.slice(1)
-//    })
-// }
+const allCaps = (array) => {
+   return array.map((string) => { 
+   return string.charAt(0).toUpperCase() + string.slice(1)
+   })
+}
```

Comment:
Your pseudo code does not match your code. You said you would be turning a string into an array and capitalizing every letter, but that's not what the code here is doing.

Reply:
It happens!